packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.spec.js

```diff
@@ -197,45 +215,90 @@ describe('Drawing tools > Mark', function () {
   })
 
   describe('useEffect hook', function () {
-    describe('with finished mark and no subTaskVisibility', function () {
-      let markWrapper = (mark) => {
-        return (
+    function markWrapper(mark) {
+      return (
+        <svg>
           <Mark
             label='Point 1'
             mark={mark}
             onDelete={onDelete}
             onFinish={onFinish}
             onSelect={onSelect}
           >
-            <Point mark={point} />
+            <Point mark={mark} />
           </Mark>
-        )
-      }
+        </svg>
+      )
+    }
+
+    describe('with finished mark and no subTaskVisibility', function () {
+      let newMark
+
+      before(function () {
+        window.scrollTo = sinon.stub()
```

Comment:
We could leverage sinon's API here if it's defined originally:
```suggestion
        scrollToStub = sinon.stub(window,  'scrollTo')
```

Or otherwise store it in a variable so we can restore the original that jsdom has set.

Reply:
I'm seeing this warning in the terminal, but nothing actually crashes or fails. I should have changes up shortly.
```
Error: Not implemented: window.scrollTo
```